### PR TITLE
RavenDB-9240: Create operations client API for identities

### PR DESCRIPTION
**Pull request by garayx**

### CODE REVIEW

File: src/Raven.Client/Documents/Commands/NextIdentityForCommand.cs [hunk 1]
```diff
@@ -1,47 +0,0 @@
-﻿using System;
-using System.Net.Http;
-using Raven.Client.Http;
-using Raven.Client.Util;
-using Sparrow.Json;
-
-namespace Raven.Client.Documents.Commands
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Commands/NextIdentityForCommand.cs]: We cannot remove those. Need to stay in the same place as they were.

File: src/Raven.Client/Documents/Commands/SeedIdentityForCommand.cs [hunk 2]
```diff
@@ -1,54 +0,0 @@
-﻿using System;
-using System.Net.Http;
-using Raven.Client.Http;
-using Raven.Client.Util;
-using Sparrow.Json;
-
-namespace Raven.Client.Documents.Commands
-{
-    public class SeedIdentityForCommand : RavenCommand<long>, IRaftCommand
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Commands/SeedIdentityForCommand.cs]: We cannot remove those. Need to stay in the same place as they were.

File: src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+namespace Raven.Client.Documents.Operations.Identities
+{
+    public class SeedIdentityForOperation : IMaintenanceOperation<long>
+    {
+        private readonly string _identityName;
+        private readonly long _identityValue;
+        private readonly bool _forceUpdate;
+
+        public SeedIdentityForOperation(string identityName, long identityValue)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs]: Rename to `name` and `value` and check if name is not null or whitespace

File: src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        {
+            _identityName = identityName;
+            _identityValue = identityValue;
+            _forceUpdate = false;
+        }
+
+        public SeedIdentityForOperation(string identityName, long identityValue, bool forceUpdate)
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs]: Rename to name and value and check if name is not null or whitespace

File: src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs (hunk 3 above)
Comment (ppekrol) [reply src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs]: Can merge those two ctors? By default force update is false?

File: src/Raven.Client/Documents/Commands/NextIdentityForCommand.cs (hunk 1 above)
Comment (garayx) [reply src/Raven.Client/Documents/Commands/NextIdentityForCommand.cs]: done

File: src/Raven.Client/Documents/Commands/SeedIdentityForCommand.cs (hunk 2 above)
Comment (garayx) [reply src/Raven.Client/Documents/Commands/SeedIdentityForCommand.cs]: done

File: src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs (hunk 3 above)
Comment (garayx) [reply src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs]: done

File: src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs (hunk 4 above)
Comment (garayx) [reply src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs]: done

**Review by ppekrol — APPROVED** [review]

**Pull request merged**